app/static/js/request/main.js

```diff
@@ -0,0 +1,7 @@
+$(function() {
+    $(".disable-enter-submit").keypress(function(e){
```

Comment:
Moved from view-request-responses.js. 
REMINDER: This should probably be just `disable-enter`